DirectEmployers/MyJobs
# [PD-3675] refactor sidebar component

## What's this PR do?
Refactors the sidebar component to use class properties

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside any of the analytics apps and the code

## How should this be manually tested?
View the code or pull down the repository and go into the analytics application. Make sure the sidebar component is still working

## Any background context you want to provide?
